src/ui/Facet/FacetSettings.ts

```diff
@@ -255,6 +279,8 @@ export class FacetSettings extends FacetSort {
           .withLabel(enabledSort.label)
           .build();
 
+        this.closePopupOnLoseFocus(elem);
```

Comment:
As my previous comment, this should ideally be called only "once" on the container itself, versus once for every section.

Reply:
I agree. With `focusout`, all of those won't be necessary anymore.